utils/dvmhost/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dvmhost
+PKG_VERSION:=2025.09.03
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=e9abdf63aa656465438d59f009a83faf5ba69f1f	# R04J32 with hotfixes
+PKG_SOURCE_URL:=https://github.com/dvmproject/dvmhost
```

Comment:
You can use here PKG_SOURCE_DATE. It is quite strange that releases are tagged as "[R04J32 2025-09-02](https://github.com/DVMProject/dvmhost/releases/tag/2025-09-03)". 😢 Using something such as semantic versioning would be much better.

Reply:
Would setting that variable remove the need to manually specify the PKG_VERSION, or is it a nice-to-have? Truthfully I am not sure what that option sets.